examples/sequential/src/module.rs

```diff
@@ -23,6 +23,11 @@ static mut SEQUENTIAL_SLOTS: &[PyModuleDef_Slot] = &[
         slot: Py_mod_multiple_interpreters,
         value: Py_MOD_PER_INTERPRETER_GIL_SUPPORTED,
     },
+    #[cfg(Py_GIL_DISABLED)]
```

Comment:
on CI, this leads to:

```
        error: unexpected `cfg` condition name: `Py_GIL_DISABLED`
          --> src/module.rs:26:11
           |
        26 |     #[cfg(Py_GIL_DISABLED)]
           |           ^^^^^^^^^^^^^^^
```

Doesn't happen for me on my local 3.12 setup. Anyone have any idea what's happening?

Reply:
Looking more closely (by setting `PIP_VERBOSE=2` in the test environment for that example), it is happening but for whatever reason this warning doesn't cause `nox` to fail on my local setup but it does trigger a failure on CI. Hopefully I can figure out why depending on pyo3-ffi isn't sufficient to use the `Py_GIL_DISABLED` attribute...